RecoLocalFastTime/FTLClusterizer/interface/MTDArrayBuffer.h

```diff
@@ -22,27 +26,58 @@ class MTDArrayBuffer {
 
   inline void setSize(uint rows, uint cols);
 
+  //use SubDet to identify whether the Hit is in BTL or ETL
+  inline GeomDetEnumerators::Location SubDet(uint row, uint col) const;
+  inline GeomDetEnumerators::Location SubDet(const FTLCluster::FTLHitPos&) const;
   inline float energy(uint row, uint col) const;
   inline float energy(const FTLCluster::FTLHitPos&) const;
   inline float time(uint row, uint col) const;
   inline float time(const FTLCluster::FTLHitPos&) const;
   inline float time_error(uint row, uint col) const;
   inline float time_error(const FTLCluster::FTLHitPos&) const;
 
+  inline LocalError local_error(uint row, uint col) const;
+  inline LocalError local_error(const FTLCluster::FTLHitPos&) const;
+  inline GlobalPoint global_point(uint row, uint col) const;
+  inline GlobalPoint global_point(const FTLCluster::FTLHitPos&) const;
```

Comment:
Hi,
Yes we are using the position and error both in the global reference frame. We think the LocalError of a RecHit should have the same value with GlobalError so we just kept that without transferring to global. Is doing that incorrect? If so, do you have any suggestions on improving that?
Sorry for the inconvenience.
Thanks.

Reply:
I'm not sure how LocalError can have the same value in the global frame.
I'd expect that the BTL "local" is at a fixed radius with x along r-phi and y along z
and in the ETL it's a fixed z with x,y as is or shifted to some reference.

Reply:
Hi,
Yeah I mean that the GlobalError may differ with LocalError by how the coordinates are set but the magnitude should be the same. So I think just using LocalError should be fine since we are just doing error_local_x^2+error_local_y^2
Or do you have any suggestions on that?

BTW, I realized that I manipulated incorrectly in `MTDThresholdClusterizer.cc` when adjusting the global position of RecHits based on its longitudinal position. Previously I used bars_in_z geometry to do this so modification should be given on local y direction but now geometry is bars_in_phi so I should do modification on local x instead of local y. I will change that in the code.

Thanks!